bower.json

```diff
@@ -12,11 +12,11 @@
   ],
   "dependencies": {
     "angular": "~1.5",
-    "angular-ui-router": "~0.2.15",
-    "ngstorage": "~0.3.3",
-    "angular-material": "~1.1",
+    "angular-ui-router": "~1.0.0-alpha.3",
+    "ngstorage": "~0.3.10",
```

Comment:
@GrahamCampbell I haven't have a problem, and it's the latest version since Novermber 2015, difference beetween versions were some pulls requests, you can see [here](https://github.com/gsklee/ngStorage/releases)